parse error は解決できた時点での maimodorun への lint error:

```fish
npm run eslint

> maimodorun@0.1.8 eslint /Users/konno/programming/ofuton/maimodorun
> eslint ./extension/js/*.js


/Users/konno/programming/ofuton/maimodorun/extension/js/background.js
  24:58  error  Parsing error: Unexpected token =>

/Users/konno/programming/ofuton/maimodorun/extension/js/inject.js
  2:46  error  Parsing error: Unexpected token =>

/Users/konno/programming/ofuton/maimodorun/extension/js/maimodorun.js
  52:45  error  Parsing error: Unexpected token =>

/Users/konno/programming/ofuton/maimodorun/extension/js/popup.js
  90:39  error  Parsing error: Unexpected token =>

/Users/konno/programming/ofuton/maimodorun/extension/js/storage.js
    1:7   error  'Storage' is assigned a value but never used        no-unused-vars
   15:20  error  'Promise' is not defined                            no-undef
   55:20  error  'Promise' is not defined                            no-undef
   69:31  error  'deleteRequest' is assigned a value but never used  no-unused-vars
  108:2   error  Missing semicolon                                   semi

✖ 9 problems (9 errors, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

--- a/extension/js/storage.js
+++ b/extension/js/storage.js
@@ -105,4 +105,4 @@ const Storage = class {
     removeItem(key) {
         return this._run('delete', key, 'readwrite');
     }
-}
+};

--- a/package.json
+++ b/package.json
@@ -8,7 +8,9 @@
   },
   "devDependencies": {},
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "./node_modules/.bin/eslint ./extension/js/*.js",
+    "eslint:fix": "./node_modules/.bin/eslint --fix ./extension/js/*.js"
   },
   "repository": {
     "type": "git",
@@ -32,9 +34,5 @@
   "bugs": {
     "url": "https://github.com/ofuton/maimodorun/issues"
   },
-  "homepage": "https://github.com/ofuton/maimodorun#readme",
-  "scripts": {
-    "eslint": "./node_modules/.bin/eslint ./extension/js/*.js",
-    "eslint:fix": "./node_modules/.bin/eslint --fix ./extension/js/*.js"
-  }
+  "homepage": "https://github.com/ofuton/maimodorun#readme"
 }

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/ofuton/maimodorun#readme",
   "scripts": {
-    "eslint": "./node_modules/.bin/eslint ./extension/js/*.js"
+    "eslint": "./node_modules/.bin/eslint ./extension/js/*.js",
+    "eslint:fix": "./node_modules/.bin/eslint --fix ./extension/js/*.js"
   }
 }

